contrib/ray/test.sh

```diff
@@ -2,46 +2,30 @@
 
 set -euxo
 
-NAMESPACE=kubeflow
+NAMESPACE=$1
 TIMEOUT=120  # timeout in seconds
 SLEEP_INTERVAL=30  # interval between checks in seconds
 RAY_VERSION=2.23.0
 
-function trap_handler {
-  kill $PID
-  # Delete RayCluster
-  kubectl -n $NAMESPACE delete -f raycluster_example.yaml
-
-  # Wait for all Ray Pods to be deleted.
-  start_time=$(date +%s)
-  while true; do
-    pods=$(kubectl -n $NAMESPACE get pods -o json | jq '.items | length')
-    if [ "$pods" -eq 1 ]; then
-      break
-    fi
-    current_time=$(date +%s)
-    elapsed_time=$((current_time - start_time))
-    if [ "$elapsed_time" -ge "$TIMEOUT" ]; then
-      echo "Timeout exceeded. Exiting loop."
-      exit 1
-    fi
-    sleep $SLEEP_INTERVAL
-  done
-
-  # Delete KubeRay operator
-  kustomize build kuberay-operator/base | kubectl -n $NAMESPACE delete -f -
-}
-
-trap trap_handler EXIT
+while [[ $(kubectl get namespace $NAMESPACE --no-headers 2>/dev/null | wc -l) -eq 0 ]]; do
```

Comment:
No while loops, this can run indefinitely. Please use a for loop with a maximum time